page.go

```diff
@@ -117,7 +118,9 @@ func (n *leafPageElement) key() []byte {
 // value returns a byte slice of the node value.
 func (n *leafPageElement) value() []byte {
 	buf := (*[maxAllocSize]byte)(unsafe.Pointer(n))
-	return (*[maxAllocSize]byte)(unsafe.Pointer(&buf[n.pos+n.ksize]))[:n.vsize]
+	ret := (*[maxAllocSize]byte)(unsafe.Pointer(&buf[n.pos+n.ksize]))[:n.vsize]
```

Comment:
I always forget about the third slice parameter. :)

Fixed in 2b4a0e28943f2dc71b7697cb86764b05e9919637.